cdc/processor/pipeline/table_actor.go

```diff
@@ -158,9 +160,13 @@ MSG:
 				t.stopFunc(err)
 			}
 		case message.TypeTick:
-			_, err := t.sinkNode.HandleMessage(ctx, pipeline.TickMessage())
-			if err != nil {
-				t.stopFunc(err)
+			// tick message flush the raw event to sink, follow the old pipeline implementation, batch flush the events  every 500ms
+			if time.Since(t.lastFlushTime) > 500*time.Millisecond {
```

Comment:
It is better to use a const var instead of `500 * time.Millisecond`.

Reply:
done